cinema/views.py

```diff
@@ -113,7 +113,7 @@ def get_serializer_class(self):
         methods=["POST"],
         detail=True,
         url_path="upload-image",
-        permission_classes=(IsAdminUser,),
+        permission_classes=(IsAdminOrIfAuthenticatedReadOnly,),
```

Comment:
you don't need to add `permission_classes`, since you've already configured them as a basic auth in setting.py